src/tutorials/konnector/index.md

```diff
@@ -5,6 +5,11 @@ To protect your data, each connector runs inside a container in order to sandbox
 
 > ⚠️ For historical reasons, in the Cozy codebase, a cozy connector is named konnector, please follow this convention if modifying an existing application.
 
+What will you need to start ?
+
+- [Node](https://nodejs.org/en/) (16), follow the link to [nodejs](https://nodejs.org/en/docs/) doc for proper installation. When it's done, you can check your version with `node --version` in your shell.
+- [Yarn](https://yarnpkg.com/), again follow the [Yarn doc](https://yarnpkg.com/getting-started) for proper install. Check the version with `yarn --version` in your shell.
```

Comment:
https://classic.yarnpkg.com/lang/en/ this is the right link for Yarn since we don't use yarn 2 ATM

Reply:
Changes done ✅